analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/AnalysisVisitor.java

```diff
@@ -432,4 +433,13 @@ public List<Description> visit(JavadocComment n, Analyzer arg) {
             !commentParams.isEmpty() ? commentParams : null,
             !commentTypeParams.isEmpty() ? commentTypeParams : null));
   }
+
+  /** Describes a Variable Declaration Expression. As these are not supported in the JSON, there is
+   * no matching description. This method is explicitly implemented to return null in order to
+   * prevent annotations being visited that are attached to the Variable Declaration.
+   */
+  @Override
+  public List<Description> visit(VariableDeclarationExpr n, Analyzer arg) {
+    return null;
```

Comment:
Prefer returning `List.of()` instead of `null` for consistency with `visit(CatchClause n, Analyzer arg)` - or, alternatively, changing that method to also return `null` in its special case.